frontend/src/components/FieldList.tsx

```diff
@@ -34,6 +34,7 @@ export const FieldListField = ({
     <FormGroup isRequired={options.isRequired} label={options.label}>
       <ComponentField
         aria-label={`Field list ${options.key}`}
+        data-testid={`field ${options.key}`}
```

Comment:
Shouldn't this include `-` in between?

Reply:
I think both of them works well
@christianvogt ??

Reply:
There's some updated guidance here: https://github.com/opendatahub-io/odh-dashboard/blob/main/docs/testing.md#selectors-and-test-ids

Space separation is correct because we now support multiple IDs being space separated in as a `data-testid`. Each ID is individually queryable.